src/markdown/tutorial/part-1/07-reusable-components.md

```diff
@@ -56,6 +56,16 @@ yarn ember test
 git add config/environment.js
 ```
 
+> Zoey says...
+>
+> The above described way of storing your MapBox token will expose it on the repository once you upload it. There is a safer way to store your public token, using dotenv:
```

Comment:
@jsantos42, thank you for your contribution! 

We agree that users should not be committing access tokens to version control, but we think that identifying the solution might over complicate things here for new users. 

Can you please try to make this a more generalized suggestion about not committing access tokens to git and pushing them to Github? 👍

Reply:
Yeah I think this is too much for the purpose of the tutorial. I also don't think committing this particular token to Git is actually a big deal, it's not really a secret, it's more like a username than a password – the full token is going to be in every API request in plaintext _served to your end-users_ anyway, leaking it via version control is the last thing you have to worry about here.

Reply:
@jaredgalanis is it better? I also corrected some linting errors.

Reply:
Hey guys! Any update here?

Reply:
I just see your change :) This looks great

Reply:
Ups! I missed that one, thank you for the grammar correction @MinThaMie!